src/Timers.h

```diff
@@ -102,6 +104,7 @@ class Timers
 
   // functions
   std::vector<Timer> LoadTimers();
+  bool findTagInTimerTags(std::string tag, std::string tags);
```

Comment:
Looking at the implementation, this method could be `static`.

Reply:
Done